src/components/DetailsInputs/DetailsInputs.js

```diff
@@ -95,7 +95,7 @@ const DetailsInputs = ({ inputs }) => {
                 key,
                 value,
                 ui: {
-                  artifactLink: generateArtifactLink(artifacts[0]),
+                  artifactLink: artifacts[0] ? generateArtifactLink(artifacts[0]) : '',
```

Comment:
return an empty string from the function instead of handling it in every place where you'll call the func